app/views/tagging_spreadsheets/show.html.erb

```diff
@@ -19,20 +15,36 @@
 
 <% else %>
 
-  <div class='view-on-site'>
-    View: <%= link_to "inspect spreadsheet", @tagging_spreadsheet.url, target: "_blank" %>
-  </div>
+  <div class="import-preview">
+    <div class='view-on-site'>
+      View: <%= link_to "inspect spreadsheet", @tagging_spreadsheet.url, target: "_blank" %>
+    </div>
 
-  <table>
-    <tr>
-      <th>Page</th>
-      <th>Will be tagged to</th>
-    </tr>
-    <% @tag_mappings.each do |tag| %>
+    <div class="progress-counter">
+      <p>
+        Create tags progress: <%= @tag_mappings.publish_confirmed.count %> of <%= @tag_mappings.count %>
+      </p>
+    </div>
+
+    <table>
       <tr>
-        <td><%= link_to tag.content_base_path, website_url(tag.content_base_path) %></td>
-        <td><%= link_to tag.link_title, content_path(tag.link_content_id) %> (<%= tag.link_type %>)</td>
+        <th>Page</th>
+        <th>Will be tagged to</th>
+        <th>Tag created?</th>
       </tr>
-    <% end %>
-  </table>
+      <% @tag_mappings.each do |tag| %>
+        <tr>
+          <td><%= link_to tag.content_base_path, website_url(tag.content_base_path) %></td>
+          <td><%= link_to tag.link_title, content_path(tag.link_content_id) %> (<%= tag.link_type %>)</td>
+          <td class="tag-mapping-status">
+            <% if tag.publish_confirmed? %>
+              <span class="label label-success">Yes</span>
+            <% else %>
+              <span class="label label-danger">No</span>
```

Comment:
Red means it's waiting for the publishing right? Perhaps `label-default` would be better. `label-danger` can then be used if the job can't be completed/errors.